runtimes/adapters/internal_agent.py

```
@@ -415,19 +415,55 @@ async def execute(self, spec: TaskSpec) -> TaskResult:
-            except Exception as exc:  # pragma: no cover - best-effort
-                self._log.debug("E2B seed failed (best-effort): %s", exc)
+                except Exception as exc:
+                    self._log.warning(
+                        "E2B clone_repo failed (falling back to seed_from_worktree): %s",
+                        exc,
+                    )
+                    # Fall back to seeding from the host worktree
+                    try:
+                        await _e2b_session.seed_from_worktree(worktree_path)
+                    except Exception:
+                        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)